components/user/community-member-card.tsx

```diff
@@ -30,9 +28,8 @@ function CommunityMemberCard({
           <Text size="xs" className="text-secondary-color">
             {address.substring(0, 10)}
           </Text>
-          <Text size="xs" className="text-secondary-color mt-1">
-            {description ?? "No description yet"}
-          </Text>
+
+          {/* TODO: Show bio properly parsed as markdown */}
```

Comment:
should we really display bio in the member cards ?
i think displayName + roles + avatar are enough isn't ?